src/RwaFlipper.sol

```diff
@@ -16,22 +19,36 @@ contract RwaFlipper {
     }
 
     VatLike public vat;
+    CatLike public cat;
     bytes32 public ilk;
     uint256 public kicks = 0;
 
 
     // --- init ---
-    constructor(address vat_, bytes32 ilk_) public {
+    constructor(address vat_, address cat_, bytes32 ilk_) public {
         vat = VatLike(vat_);
+        cat = CatLike(cat_);
         ilk = ilk_;
         wards[msg.sender] = 1;
     }
 
-    function kick(address usr, address gal, uint tab, uint lot, uint bid) public auth returns (uint id) {
+    function file(bytes32 what, address data) external note auth {
+        if (what == "cat") cat = CatLike(data);
+        else revert("RwaFlipper/file-unrecognized-param");
+    }
+
+    function kick(
+        address usr,
+        address gal,
+        uint tab,
+        uint lot,
+        uint bid
+    ) public auth returns (uint id) {
```

Comment:
Can be `external` instead of `public`, I believe.

Reply:
I think this is just to be conformant with our other `flip,kick()` interfaces